.github/workflows/netlify-build.yaml

```diff
@@ -1,22 +0,0 @@
-name: Netlify deploy hook to add new docs on Redwoodjs.com
-
-on:
-  push:
-    branches:
-      - main
-    paths:
-      - '**.md'
-  release:
-    types:
-      - published
-      - edited
```

Comment:
I don't have a holistic picture of all events currently trigger docs build but I thought this triggers on every release, so the version is updated to the latest on docs site. Is there any other event that does now?

Reply:
@jtoar Even though we don't have docs to publish, we do use this Action to trigger publishing in order to update the RW Version in the header.

_However,_ I don't think it works... Because it's dynamic, I've observed this action running after a new release but the corresponding published rwjs.com does not have an updated version. So I'll manually trigger a build+deploy on Netlify and the version will then be updated. My guess is that this is do to Netlify build attempting to be smart about performance and only build pages/content that changed. 

Recommendation:
- is there a way to call a build hook that forces rebuild of everything?
- if not, this Action is useless as is and should be removed